src/main/java/frc/utils/applicationsutils/FileCreator.java

```diff
@@ -0,0 +1,85 @@
+package frc.utils.applicationsutils;
+
+import frc.robot.constants.DirectoryPathsConstants;
+
+import java.awt.*;
+import java.io.File;
+import java.io.FileWriter;
+
+public class FileCreator {
+
+    public static final File OUTPUT_FILE = DirectoryPathsConstants.OUTPUT_FILES_DIRECTORY_PATH.resolve("FileCreator.txt").toFile();
+
+    public static File createFile(File file) {
+        try {
+            file.createNewFile();
+            return file;
+        }
+        catch (Exception exception) {
+            writeToOutputFile(OUTPUT_FILE, "\nException while creating file " + file + "\n" + exception, true);
+            return null;
+        }
+    }
+
+    public static void writeToTextFile(File file, String text, boolean isAppending) {
+        try {
+            FileWriter myWriter = new FileWriter(file,isAppending);
+            myWriter.write(text);
+            myWriter.close();
+        }
+        catch (Exception exception) {
+            writeToOutputFile(OUTPUT_FILE, "\nException while writing to text file " + file + "\n" + exception, true);
```

Comment:
Make the appending const and use it once, it will be a convention to write to file with or without append